app/assets/javascripts/admin/addon/models/site-setting.js

```diff
@@ -47,6 +55,10 @@ export default class SiteSetting extends EmberObject {
     });
   }
 
+  @tracked isSaving = false;
+  @tracked validationMessage = null;
+  updateExistingUsers = false;
```

Comment:
State moved to the model so that it can:

1) Persist when the component is destroyed.
2) Be shared between both `SiteSettingComponent` and `SiteSettingChangeTracker`.